cluster-provision/centos8/scripts/vm.sh

```diff
@@ -119,4 +129,4 @@ exec qemu-system-x86_64 -enable-kvm -drive format=qcow2,file=${next},if=virtio,c
   -serial pty -M q35,accel=kvm,kernel_irqchip=split \
   -device intel-iommu,intremap=on,caching-mode=on -soundhw hda \
   -uuid $(cat /proc/sys/kernel/random/uuid) \
-  ${QEMU_ARGS}
+  ${QEMU_ARGS} ${NVME_QEMU_ARG}
```

Comment:
Why do we need to touch vm.sh ? The NVMe qemu arguments must be after the other arguments ? If not this can be implemented just at gocli passing the NVMe arguments to QEMU_ARGS.

Reply:
@qinqon we need to create the image before calling qemu. For the rest, you are right I can pass the arguments as extra qemu args. I can also move the image creation in gocli

Reply:
@alicefr I think there is a pretty simple way to still do this at `cluster-up` in contrast to `cluster-provision` while keeping most of your changes like they are. The `vm.sh` script is executed in both phases. You could move the `gocli` flag from `provision` to `up` and the disks would then only be created when `vm.sh` is executed for `cluster-up`. Similar like you can sets different memory for  both phases.

Reply:
sorry @rmohr I don't quite understand the comment. Do you mean moving the `--nvme` flag from `gocli run` to `gocli provision`?

Reply:
No. Forget my comment. Everything is where it should be. I hod for some reason the impression that you provision the images during the provision phase. :+1: